src/components/articles/AllArticlesCard.jsx

```diff
@@ -0,0 +1,17 @@
+import "./AllArticles.css";
+
+export default function AllArticlesCard({ article }) {
```

Comment:
I think this component's a bit confusingly named as it describes one ArticleCard, but I see what you're getting at
Unless you mean to have a different card component for when it's displaying AllArticles as opposed to just the article cards for a certain topic? If this is the case consider whether this is necessary, or whether it would make sense to have a more generic reusable ArticleCard component